typescript/web/src/keymap.ts

```diff
@@ -23,6 +23,11 @@ export const keymap: Keymap = {
     description: "Polygon tool",
     category: "Tools",
   },
+  toolFreehand: {
+    key: "h",
+    description: "Freehand tool",
```

Comment:
@nicOwlas is the `h` shorcut OK to you?
_(I'd use `f` but it's quite subjective)_

Reply:
I would have too, but already taken by `Enter full screen` 😕

Reply:
> F11 for windows

Yes but on most laptops, the F keys are now mapped some hardware features by default (e.g. volume, brightness, etc). Unless you change some settings in the BIOS, it's needed to also press the `Fn` key.

So `f` is often used as an alternative on some websites like YouTube...

Reply:
I did not know that `f` was also working on Youtube!

Lets keep `h` for freehand, it's still easy enough to remember.